lib/phoenix_html/form_data.ex

```diff
@@ -91,6 +91,7 @@ defimpl Phoenix.HTML.FormData, for: [Plug.Conn, Atom] do
     {append, opts} = Keyword.pop(opts, :append, [])
     {name, opts} = Keyword.pop(opts, :as)
     {id, opts} = Keyword.pop(opts, :id)
+    {hidden, opts} = Keyword.pop(opts, :hidden, [])
```

Comment:
@josevalim I'm just passing this as an option rather than creating a new implementation, not sure if that's a good idea though 🙈. Could you review that? if it doesn't, I will try to figure out how to creating a new implementation of `FormData` to use only in the tests.